index.html

```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <h3 class="title2">Multi-Post Stories</h3>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
+        <div class="items buttontypeA">
+        <a href="#">See Project</a>
+        </div>
+        <div class="items site1">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+            </div>
+        </div><!--site1 end-->
+        <br> <br> <br>
+        <div class="items site1 site2">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</section><!--page2 end-->
+<br> <br> <br>
+<section class="page3">
+    <div class="container">
+        <br> <br> 
+        <h3 class="items title1">About me</h3>
+        <p class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="items buttontypeA">
+            <a href="#">Get My Resume</a>
+        </div>
+        <div class="items graphic">
+            <img src="images/graphic.png" alt="">
+        </div>
```

Comment:
[OPTIONAL] Let's try to fix these design issues. You seemingly forgot to use the same padding and margins as specifiedd in the Figma file.
## Your Design
![image](https://user-images.githubusercontent.com/45668340/148455294-ec369479-86d5-446d-896e-7889eb30a239.png)

## Figma Design
![image](https://user-images.githubusercontent.com/45668340/148455505-07f5f224-73bf-4137-8eb8-645363d8c5df.png)
Please, try to space out the button with the same margin as we have it in the Figma file.